pages/common/bazel.md

```diff
@@ -0,0 +1,28 @@
+# bazel
+
+> Open-source build and test tool similar to Make, Maven, and Gradle.
+> More information: <https://bazel.build/reference/command-line-reference>.
+
+- Show available commands and usage:
```

Comment:
Can you move this `help` command to last?

Reply:
Done.